tests/test_views.py

```diff
@@ -687,8 +688,12 @@ def test_neighborhoods_view(self):
 
         # Check that the rendered context contains the list of neighborhoods.
         self.assertIn("neighborhoods", response.context)
-        neighborhoods = response.context["neighborhoods"]
-        self.assertEqual(neighborhoods.count(), 1)
+        json_neighborhoods = response.context["neighborhoods"]
+        neighborhoods = json.loads(json_neighborhoods)
+        tp = neighborhoods[0]["fields"]
+        neighborhood = Neighborhood.objects.all()
+        p = neighborhood[0].name
+        self.assertEqual(p, tp["name"])
```

Comment:
Please revert this change. The aim of the test is to check the the neighborhoods view and the count is just fine. I would suggest we keep the tests similar.